db/sqldb.go

```diff
@@ -132,7 +132,7 @@ func (db *SQLDatabase) GetMTASTSStats(source string) (stats.Series, error) {
 		if err := rows.Scan(&a.Time, &a.WithMXs, &a.MTASTSTesting, &a.MTASTSEnforce); err != nil {
 			return stats.Series{}, err
 		}
-		series[a.Time.UTC()] = a.PercentMTASTS()
+		series[a.Time.UTC()] = float64(a.TotalMTASTS())
```

Comment:
why does this have to be a float?

Reply:
The underlying type of `Series` (which this fn returns) is `map[time.Time]float64`

Reply:
The underlying type of `Series` (which this fn returns) is `map[time.Time]float64`. That makes it also able to hold a series of percents, as in `GetMTASTSLocalStats`.

This inconsistency gets cleaned up in PR #239, which changes the underlying type of `Series` to `[]checker.AggregatedScan`

Reply:
Ooh okay I see! Thanks.